src/plugins/vis_types/timelion/public/components/timelion_expression_input.tsx

```diff
@@ -83,11 +83,17 @@ function TimelionExpressionInput({ value, setValue }: TimelionExpressionInputPro
   );
 
   useEffect(() => {
+    const abortController = new AbortController();
```

Comment:
I don't think we need an abort handler for this request - this implementation is extremely simple and only returning an object out of JS memory: https://github.com/elastic/kibana/blob/e20f530762729a8b78a31a1e8b19845bcb83e730/src/plugins/vis_types/timelion/server/routes/functions.ts#L25

It's not hitting any upstream services

Reply:
Yes, on the other hand, it's just good practice to cancel requests that we no longer need, if you think it's not necessary, let's delete it, but I don't see a problem here